src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -56,23 +56,35 @@ public void storeMessage(Message message) {
    * @return a list of messages received by the user, or empty list if user has never received a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessagesByRecipient(String recipient) {
+
+  public List<Message> getAllMessages() {
     List<Message> messages = new ArrayList<>();
+    Query query = new Query("Message")
+        .addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+    return getMessages(results);
+  }
 
-    Query query =
-        new Query("Message")
-            .setFilter(new Query.FilterPredicate("recipient", FilterOperator.EQUAL, recipient))
-            .addSort("timestamp", SortDirection.DESCENDING);
+  /* Will get messages filtered by a specific recipient */
```

Comment:
Javadoc comments have to start with two stars `/**` (but still end with the normal `*/`)